lib/src/realm_class.dart

```diff
@@ -52,19 +52,24 @@ class Realm {
 
   /// Opens a `Realm` using a [Configuration] object.
   Realm(Configuration config) : _config = config {
-    _scheduler = Scheduler(config, close);
+    if (_config.isInUse) {
+      throw RealmException("A Realm instance for this configuraiton object already exists.");
```

Comment:
I think this should be `RealmStateError`. since this is not something people should catching and handling.

Reply:
Done